<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>375</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #202932;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4">
         <widget class="QCheckBox" name="start_lock">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>开启锁人</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_6">
          <property name="toolTip">
           <string>范围0~10,建议1</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>平滑度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="plan">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <item>
           <property name="text">
            <string>长按</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>切换</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QDoubleSpinBox" name="smooth">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="maximum">
           <double>10.990000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="iou">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.850000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="4">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="headtofoot">
            <property name="cursor">
             <cursorShape>SizeHorCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">/*滑块的样式*/
QSlider::groove:horizontal {
border: 1px solid #00B0AE;
background: #00000000;
height: 2px;
border-radius: 8px;
padding-left:0px;
padding-right:0px;
}

/*滑块经过的颜色:前面的颜色*/
QSlider::sub-page:horizontal {
background: #00B0AE;
border: 1px solid #00B0AE;
height: 2px;
border-radius: 2px;
}

QSlider::add-page:horizontal {
background: #EAEAEA;
border: 0px solid #EAEAEA;
height: 2px;
border-radius: 2px;
}
QSlider::handle:horizontal 
{
    background: #99efdc;
    width: 15px;
    margin-top: -6px;
    margin-bottom: -6px;
	border-radius:16;

}

QSlider::handle:horizontal:hover {
    background: #55e5c5;

    width: 15px;
    margin-top: -6px;
    margin-bottom: -6px;
    border-radius: 16px;
	border:16px;
}




</string>
            </property>
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>置信阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>瞄准模式</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="4">
         <widget class="QDoubleSpinBox" name="y_value">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.350000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="mess">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>使用mess截屏</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color:white; font-family:Microsoft YaHei;</string>
            </property>
            <property name="text">
             <string>准星位置</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="12" column="2">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>y:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QComboBox" name="mouse">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <item>
           <property name="text">
            <string>右键</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左键</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="aim_mode">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <item>
           <property name="text">
            <string>身体</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>头</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>鼠标</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>交并比阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="1">
         <widget class="QDoubleSpinBox" name="x_value">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.180000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="debug">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>调试模式</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="value_belive">
          <property name="styleSheet">
           <string notr="true">color:white;</string>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.250000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>触发方式</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="cuda">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>使用Cuda</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>x:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color:white; font-family:Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>截屏占比：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButton:normal/*鼠标正常时的效果*/
{
    color:#000000;
    background-color:rgb(153, 239, 220); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:4px;
    padding-top:4px;
}
 
 /*鼠标悬浮时的效果*/
QPushButton#pushButton:hover
{
    color:#000000;
    background-color:rgb(85, 229, 197); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
}

/*如果按下与悬浮想同时产生效果，hover必须写在pressed的后面*/
/*鼠标按下时的效果*/
QPushButton#pushButton:pressed
{
    color:#000000;
    background-color:rgb(85, 229, 197); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:6px;
    padding-top:6px;
}

/*鼠标不可用时的效果*/
QPushButton#pushButton:disabled
{
    color:#000000;
    background-color:rgb(40, 85, 20); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:6px;
    padding-top:6px;
}
 
QPushButton
{
    color:#000000; /*文字颜色*/
    background-color:rgb(153, 239, 220);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:rgb(29, 38, 47); /*边框颜色*/
    border-radius:15px; /*边框倒角*/
    font:bold 14px; /*字体*/
    font-family: Segoe UI;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
 

</string>
          </property>
          <property name="text">
           <string>确认修改</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exit_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton:normal/*鼠标正常时的效果*/
{
    color:#000000;
    background-color:rgb(153, 239, 220); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:4px;
    padding-top:4px;
}
 
 /*鼠标悬浮时的效果*/
QPushButton:hover
{
    color:#000000;
    background-color:rgb(85, 229, 197); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
}

/*如果按下与悬浮想同时产生效果，hover必须写在pressed的后面*/
/*鼠标按下时的效果*/
QPushButton:pressed
{
    color:#000000;
    background-color:rgb(85, 229, 197); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:6px;
    padding-top:6px;
}

/*鼠标不可用时的效果*/
QPushButton:disabled
{
    color:#000000;
    background-color:rgb(40, 85, 20); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:6px;
    padding-top:6px;
}
 
QPushButton
{
    color:#000000; /*文字颜色*/
    background-color:rgb(153, 239, 220);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:rgb(29, 38, 47); /*边框颜色*/
    border-radius:15px; /*边框倒角*/
    font:bold 14px; /*字体*/
    font-family: Segoe UI;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
 
</string>
          </property>
          <property name="text">
           <string>进程退出</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>MainWindow</sender>
   <signal>destroyed()</signal>
   <receiver>exit_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
